.github/rd-release-config.yml

```diff
@@ -0,0 +1,167 @@
+filter-by-commitish: False
+include-pre-releases: false
+categories:
+  - title: '🔥 Features'
+    labels:
+      - 'features'
+  - title: '🚀 Enhancemets'
+    labels:
+      - 'performance'
+      - 'enhancement'
+  - title: '🐛 Bug Fixes'
+    labels:
+      - 'bug'
+      - 'bug :bug:'
+  - title: '💯 Benchmarking'
+    label: 'benchmarking'
+  - title: '📜 Documentation'
+    labels:
+      - 'documentation'
+  - title: 'Testing'
+    label: 'testing'
+  - title: '🔒 Security'
+    labels:
+      - 'security'
+  - title: 'Core'
+    label: 'core'
+  - title: 'Classification'
+    label: 'classification'
+  - title: 'Cluster'
+    label: 'cluster'
+  - title: 'Datasets'
+    label: 'datasets'
+  - title: 'Decomposition'
+    label: 'decomposition'
+  - title: 'FFT'
+    label: 'fft'
+  - title: 'Graph'
+    label: 'graph'
+  - title: 'Linear Algebra'
+    label: 'linalg'
+  - title: 'Naive Bayes'
+    label: 'naive bayes'
+  - title: 'Neural Networks'
+    labels:
+      - 'nn'
+      - 'neural net'
+  - title: 'Optimization'
+    label: 'optim'
+  - title: 'Preprocessing'
+    label: 'preprocessing'
+  - title: 'Regression'
+    label: 'regression'
+  - title: 'Sparse'
+    label: 'sparse'
+  - title: 'Spatial'
+    label: 'spatial'
+  - title: 'Utils'
+    label: 'utils'
+exclude-labels:
+  - 'github-actions'
+  - 'dependencies'
+  - 'chore'
+  - 'workflows'
+
+autolabeler:
```

Comment:
When is this action triggered? Ideally PRs should be labeled when they are first opened.